.github/workflows/.windows.yml

```diff
@@ -428,7 +428,7 @@ jobs:
         name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: ${{ env.GO_VERSION }}
+          go-version: 1.21.12
```

Comment:
Oh, wait; why was this one changed from the env-var?

Reply:
Huh, looks like I borked something. I'll open a follow up PR.

Reply:
https://github.com/moby/moby/pull/48121

Reply:
Thanks! I completely missed it here, but noticed it when I rebased my "go1.23" PR 😂